src/client.js

```diff
@@ -81,7 +83,7 @@ export class Saturn {
       }
     }
 
-    const origins = options.origins
+    const origins = options.origins || [options.cdnURL]
```

Comment:
if `options.origins` is an empty array it's still truthy, is that ok here?

Reply:
not ok, thanks for flagging. I've added more logic to handle this better.